packages/api/src/app-router.ts

```diff
@@ -268,10 +266,19 @@ export default async function makeApp(params: CliArgs) {
   if (fallbackProxy) {
     app.use(proxy({ target: fallbackProxy, changeOrigin: true }));
   }
+
+  let wwwHandler;
   if (frontend) {
-    const wwwHandler = await createFrontend();
-    app.use(wwwHandler);
+    wwwHandler = await createFrontend();
   }
+  const errHandler = errorHandler();
+
+  app.use((err, _req, res, _next) => {
+    if (frontend && !_req.path.startsWith(httpPrefix)) {
+      wwwHandler(_req, res, _next);
```

Comment:
@iameli I'm not sure this works, because I don't know in which scenarios it is used 🙃 Could you double-check?

Reply:
it's being used to setup a proxy from api server to frontend for `livepeer-in-a-box` project.

Reply:
Will this work after my change? 🙃

Reply:
should be easy to verify by updating the manifest file for that project to your commit in this project. once the build pipeline finishes: https://github.com/livepeer/studio/actions/runs/5938452782/job/16102930628?pr=1836

Reply:
*branch + commit sha

Reply:
You probably want to check if `err` is null, otherwise this could eat api handler exceptions afaik 

It would be weird for this to be called without an error tho as per the express error handler instructions